cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
+  "project_slug": "django_drf_boilerplate",
+  "project_title": "Django DRF Boilerplate",
+  "enable_sentry": "y",
+  "enable_health_check": "n",
+  "use_git": "y",
+  "use_circleci": "y"
```

Comment:
```suggestion
  "enable_sentry": true,
  "enable_health_check": false,
  "use_git": true,
  "use_circleci": true
```

Reply:
Cookiecutter doesn't work well if bools :( https://github.com/cookiecutter/cookiecutter/pull/548

Reply:
Unless you're planning for more than 2 options on each of those... 🤔

Reply:
> Cookiecutter doesn't work well if bools :( cookiecutter/cookiecutter#548

Oh 😢

Reply:
Unreliable at the moment, see: https://github.com/cookiecutter/cookiecutter/pull/848